maintenance_mode/views.py

```diff
@@ -24,4 +24,4 @@ def maintenance_mode_on(request):
     if request.user.is_superuser:
         set_maintenance_mode(True)
 
-    return HttpResponseRedirect('/')
+    return HttpResponseRedirect(request.META['SCRIPT_NAME'] or '/')
```

Comment:
`request.META` is a dictionary, a better and `KeyError` proof approach would be to use the `dict.get` method with a default value: 
```python
return HttpResponseRedirect(request.META.get('SCRIPT_NAME', '/'))
```

Reply:
As you can see, the redirect response contains some custom logic and it is repeated twice.

Reply:
Yes, this makes much sense. I will convert them.